runtime/scripts/exam.js

```diff
@@ -576,6 +576,7 @@ Exam.prototype = /** @lends Numbas.Exam.prototype */ {
     /** Restore previously started exam from storage.
      *
      * @fires Numbas.Exam#event:ready
+     * @fires Numbas.Part#event:loaded
```

Comment:
This should be `Numbas.Exam`. Can you check the rest of the comments you've added, too?